### PR TITLE
(MODULES-3218) add auth_merging for directory enteries

### PR DESCRIPTION
**Pull request by pyther**

### CODE REVIEW

**Comment by hunner** [comment]
Could you rebase this? That will hopefully make the tests go green.

File: README.md [hunk 1]
```diff
@@ -2804,6 +2804,10 @@ Sets the value for [AuthType](https://httpd.apache.org/docs/current/mod/mod_auth
 
 Sets the value for [AuthUserFile](https://httpd.apache.org/docs/current/mod/mod_authn_file.html#authuserfile), which sets the name of the text file containing the users/passwords for authentication.
 
+###### `auth_merging`
+
+Sets the value for [AuthMerging](https://httpd.apache.org/docs/current/mod/mod_authz_core.html#authmerging), which determines if authorization logic should be comibend
```
Comment (bmjen) [inline README.md]: comibend => combined?

Comment (hunner) [inline README.md]: Typo `combined`

**Comment by bmjen** [comment]
Please rebase from master to get rid of the Travis-CI failure.

**Comment by bmjen** [comment]
Thanks for the contribution @pyther !

**Pull request merged**